public/js/index.js

```diff
@@ -1769,40 +1768,46 @@ socket.on('version', function (data) {
     }
   }
 })
-var authors = []
-var authorship = []
-var authorMarks = {} // temp variable
-var addTextMarkers = [] // temp variable
+let authors = []
+let authorship = []
+let authorMarks = {} // temp variable
+let addTextMarkers = [] // temp variable
 function updateInfo (data) {
   // console.debug(data);
+  // eslint-disable-next-line no-prototype-builtins
   if (data.hasOwnProperty('createtime') && window.createtime !== data.createtime) {
```

Comment:
The [suggested way](https://eslint.org/docs/rules/no-prototype-builtins#rule-details) to write this without the eslint warning would be:
`Object.prototype.hasOwnProperty.call(data, 'createtime')`

I don't know whether it's important to change it here, so decide for your self whether you want to change this.

Reply:
fixed